src/redux/reducers/credentialsReducer.js

```diff
@@ -1,9 +1,24 @@
 import { credentialsTypes } from '../constants';
-import { helpers } from '../../common/helpers';
-import { reduxHelpers } from '../common/reduxHelpers';
-import apiTypes from '../../constants/apiConstants';
+import { reduxHelpers } from '../common';
+import { helpers } from '../../common';
+// import apiTypes from '../../constants/apiConstants';
 
 const initialState = {
+  // confirmDelete: {},
```

Comment:
required:
remove these commented lines, `4` and `7`, and the below commented block